frontend/public/components/storage-class.tsx

```diff
@@ -38,120 +39,125 @@ const tableColumnClasses = [
   Kebab.columnClass,
 ];
 
-const StorageClassTableHeader = () => {
-  return [
-    {
-      title: 'Name',
-      sortField: 'metadata.name',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[0] },
-    },
-    {
-      title: 'Provisioner',
-      sortField: 'provisioner',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[1] },
-    },
-    {
-      title: (
-        <>
-          Reclaim <span className="hidden-sm">Policy</span>
-        </>
-      ),
-      sortField: 'reclaimPolicy',
-      transforms: [sortable],
-      props: { className: tableColumnClasses[2] },
-    },
-    {
-      title: '',
-      props: { className: tableColumnClasses[3] },
-    },
-  ];
+const StorageClassDetails: React.FC<StorageClassDetailsProps> = ({ obj }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('public~StorageClass details')} />
+        <div className="row">
+          <div className="col-sm-6">
+            <ResourceSummary resource={obj}>
+              <DetailsItem label={t('public~Provisioner')} obj={obj} path="provisioner" />
+            </ResourceSummary>
+          </div>
+          <div className="col-sm-6">
+            <dl className="co-m-pane__details">
+              <DetailsItem label={t('public~Reclaim policy')} obj={obj} path="reclaimPolicy" />
+              <dt>{t('public~Default class')}</dt>
+              <dd>{isDefaultClass(obj).toString()}</dd>
```

Comment:
True/False should probably be translated like we do here:

https://github.com/openshift/console/blob/master/frontend/public/components/machine-config-pool.tsx#L104

Reply:
updated